src/DIRAC/WorkloadManagementSystem/OptimizerAdministrator/OptimizerAdministrator.py

```diff
@@ -0,0 +1,88 @@
+"""
+The OptimizerAdministrator determines the chain of Optimizing Agents
+that must work on the job prior to the scheduling decision and
+launches thoses agents
```

Comment:
Don't use the word "agent" here, as "agent" has a specific meaning in DIRAC. Use instead "optimization tasks"